src/module/actor/data-model-classes/data-model-character-encumbrance-complete.ts

```diff
@@ -19,6 +19,12 @@ export default class OseDataModelCharacterEncumbranceComplete
 
   static templateInventoryRow = "";
 
+  static encumbranceStepsComplete = {
```

Comment:
I remembered another reason that I'd prefer not to add different `encumbranceSteps` static members to each encumbrance scheme.

Check out [our constants file](https://github.com/vttred/ose/blob/main/src/module/config.ts#L35-L38). We export the selected encumbrance scheme as a constant. I have no clue if someone's built anything that uses this, but we expose it, and it's been exposed for some time.

Reply:
The more I think about this whole "look at what item slot encumbrance has done to my sweet boy" thing, the more I wonder if we ought to have a separate move speed class for item-based encumbrance, and select it based on the encumbrance type in use. Item-based encumbrance muddies so much!

Elegant? Maybe not. But it hides our sins, and separates system-specific stuff from what will get ripped out for UFT, once we can (eventually!) get there.

Absolutely not a "this PR" discussion, but something to chew on.